android/assets/jsons/Translations/Other.json

```diff
@@ -83,6 +86,7 @@
 		Simplified_Chinese:"回合"
 		Spanish:"Turno"
 		Czech:"Tah"
+		Ukrainian:""
```

Comment:
you have not translated it.

Reply:
I know, so I write WIP in the pull request name